whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -16970,26 +17009,75 @@
             "instanceOf": {
               "@type": "Text",
               "supplementTo": [{
-                  "@type": "Work",
-                  "hasInstance": {
-                    "@type": "Instance",
-                    "hasTitle": [{
-                      "@type": "Title",
-                      "mainTitle": "Briefec"
-                    }],
-                    "provisionActivityStatement": "provisionActivityStatement",
-                    "describedBy": [{
-                      "@type": "Record",
-                      "controlNumber": "000"
-                    }]
-                  },
-                  "marc:controlSubfield": "p1",
```

Comment:
I surmise that this was removed since it doesn't appear to have been used (in the shapes)? And/or did it cause trouble as well? (Not that I see any use for it; just to be sure.)

Reply:
It has been used though. The shapes file seems to be missing all 'marc:*' properties, for some hitherto unknown reason...

Reply:
(Correction: *my* files appear to miss them.)

Reply:
Concerning the subfields that I have made ignored=true:

I have received ok to remove from the local libraries, we have been in contact with
- marc:controlSubfield
- partNumber

Reply:
Very good! Mostly needed to update my assumptions.

Reply:
Regarding marc:fieldref and marc:groupid I only have one note that says that we have decided to remove and ignore both of them. But I can't really remember how we came to this conclusion. Do you remember @niklasl @klngwll ?
I have checked the shapes files and they do occur, but are not very common; 338 occurrences of marc:fieldref in all link fields (of which hasPart, otherPhysicalFormat, continues and relatedTo have the most occurrences) and only one occurrence of marc:groupId.